### PR TITLE
Accessibility Changes

### PR DESCRIPTION
**Pull request by mckent05**
In this pull request, we (@mckent05 and @Meri-MG) have checked and made changes to make my portfolio according to the accessibility requirement. 

The changes made are:
**Page titles**- [changed page title](https://github.com/mckent05/Portfolio_Project/compare/accessibility_branch?expand=1#diff-bd60669fce289cf66b7cc414552c07f192a0bf60d58a59cce3e14d327285b19bL20-L21)
**Image text alternatives**** (alt text) - [included descriptive alt text for img](https://github.com/mckent05/Portfolio_Project/compare/accessibility_branch?expand=1#diff-bd60669fce289cf66b7cc414552c07f192a0bf60d58a59cce3e14d327285b19bL52-R52)
**Text headings** - [h3 tag was changed to h2 according to heading rules](https://github.com/mckent05/Portfolio_Project/compare/accessibility_branch?expand=1#diff-bd60669fce289cf66b7cc414552c07f192a0bf60d58a59cce3e14d327285b19bL96-R96)
**Color contrast** - [fixed issues with button color contrast](https://github.com/mckent05/Portfolio_Project/compare/accessibility_branch?expand=1#diff-a637ada922f044a5aba3ffee2a227071c550757946a5da02b7c2c8c17e1e9fc6R418-R419)
**Resize** - No issues found
**Interaction** - No issues found
**Moving content** - No issues found
**Multimedia** - No issues found
**The basic structure of the page** - No issues found.

**Please note that this is a new pull request since the old and unapproved pull request was mistakenly merged to the main branch. We have implemented the changes requested by the previous reviewer.

You can find the link to the requested changes [here](https://github.com/mckent05/Portfolio_Project/pull/9)**

### CODE REVIEW

File: mobile.html [hunk 1]
```diff
@@ -259,9 +207,9 @@ <h2 class="project-title">Multi-Post Stories</h2>
                     </button>
```
Comment (Mihndim2020) [inline mobile.html:207]: - [ ] As requested by the previous code reviewer, which is also part of the project requirements, and best practices sticking to the design as much as possible, please kindly the hover state your buttons. The color codes for the various button states are found in the figma design. To get the color to use for the hover state of the buttons, click on the hover button in the figma design(see image below), then look at the bottom right, there you will see the various properties of the button, including the color.
- [OPTIONAL] Please make sure the colors your buttons match those in design, see images below;

***The design***
mcD
***Your implementation***  
mcI

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
## Hi @Meri-MG and @mckent05 , 

Great work on making the changes requested by a previous reviewer 👏 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
stay-strong-youre-almost-there


 # Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

## Please make sure you do not merge an unapproved PR next time.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mckent05** [comment]
Please note that the button colours in the design have been strictly followed and they might not pass the contrast test as noted by the first reviewer.

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**